docs/development-and-testing.md

```diff
@@ -0,0 +1,57 @@
+# Development and Testing
+## Running pipeline scripts locally
+The script that's run in the pipeline is located in [./hack/app_sre_build_deploy.sh], this script is used to build and deploy the operator image to the app-sre repository. We'll also detail how to build and push an image to your own quay repository.
+
+### Setting environment variables
+Make sure to have the following environment variables set:
+```
+export REGISTRY_TOKEN=... # Get from quay.io user settings "Generate encrypted password"
+export REGISTRY_USER=... # quay.io username
+export IMG=quay.io/$REGISTRY_USER/managed-node-metadata-operator
+export IMAGE_REPOSITORY=$REGISTRY_USER
+export CONTAINER_ENGINE=docker # Currently this script requires docker as container engine
+export QUAY_IMAGE=$IMG
+export QUAY_USER=$REGISTRY_TOKEN
+export QUAY_TOKEN=$REGISTRY_TOKEN
+export QUAY_USER=$REGISTRY_USER
```

Comment:
why is QUAY_USER is set $REGISTRY_TOKEN and then to $REGISTRY_USER?

Reply:
REGISTRY_USER is the right one. So it will work but line 14 shouldn't be there indeed.